includes/footer-homepage.php

```diff
@@ -0,0 +1,35 @@
+<script src="/assets/js/vendor/modernizr-2.8.3-respond-1.4.2.min.js"></script>
+<script src="/assets/js/vendor/jquery-3.0.0.min.js"></script>
+<script src="https://netdna.bootstrapcdn.com/bootstrap/3.1.1/js/bootstrap.min.js"></script>
+<script src='/assets/js/vendor/fastclick.js'></script>
+<script src='/assets/js/vendor/gridder.js'></script>
+<script src='/assets/js/vendor/buddysystem.jquery.js'></script>
+<script src='/assets/js/vendor/scrollreveal.min.js'></script>
+<script src='/assets/js/vendor/isscrolledintoview.js'></script>
+<script src='/assets/js/vendor/glide.min.js'></script>
+<script src='/assets/js/vendor/stickyfill.js'></script>
+<script src='/assets/js/app.js'></script>
+<footer id="footer">
+  <div class="footer homepage">
+    <div class="row">
+      <div class="col-md-7 footerinfo homepage">
+        <img class="footerlogo" src="assets/img/logos/fd-learning_games_logo_new_alt.png" alt="the Field Day Lab logo">
+        <p class="small">Our <a href="/about">team</a> is based at the Wisconsin Center for Education Research at the University of Wisconsin - Madison. We make learning games that bring contemporary research to the public, and we use the game data to understand how people learn.</p>
+        <p class="small">Interested in making something together? email <a href="mailto:djgagnon@wisc.edu">David Gagnon</a><br />
+        for all other inquiries, please contact <a href="mailto:rgroff@wisc.edu">Reyna Groff</a></p>
+        <div class="social-links">
+          <a target="_blank" href="http://facebook.com/fielddaylab"><img src="/assets/img/logos/social-facebook.png"></a>
+          <a target="_blank" href="http://twitter.com/fielddaylab"><img src="/assets/img/logos/social-twitter.png"></a>
+          <a target="_blank" href="http://instagram.com/fielddaylab"><img src="/assets/img/logos/social-insta.png"></a>
+        </div>
+      </div>
+      <div class="col-md-5">
+        <div class="footer-signup">
+          <img class="signup-badge img-responsive" src="/assets/img/illustrations/mask.png" alt="a stylized and colorfully patterned mask">
+          <p class="small">Interested in what we're cooking up? We have many ways for you to get involved.</p>
+           <a target="_blank" href="#" class="button xsmall">GET INVOLVED</a>
```

Comment:
Please use existing classes `white` and `filled` for the button. Those are defined in `_buttons.scss` so they can be updated globally with theme updates. I know the border radius is different from the mockup, but we should just use the theme buttons. If Eric wants to change the theme button radius he can. 
The only style change should be to add the hover effect to focus for keyboard nav.
```
a.button:focus {
  opacity: 0.9;
}
```